go.mod

```diff
@@ -1,18 +1,19 @@
 module github.com/prometheus/blackbox_exporter
 
-go 1.21
+go 1.22
+
+toolchain go1.23.1
```

Comment:
Should this just be `1.23` without specifying the toolchain?

Reply:
Thanks! We don't need to specify toolchain here, it just got added in automatically by `go mod tidy`. The project gets built by go `1.23` in CI, using `1.22` here should be fine as it's just the minimum required go version to use to build.
I can remove `toolchain` :+1: